src/api/java/CMakeLists.txt

```diff
@@ -9,8 +9,55 @@
 ## directory for licensing information.
 ##
 
-# create a directory for the cvc package
-file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/cvc")
+# create directories
+file(MAKE_DIRECTORY
+  "${CMAKE_CURRENT_BINARY_DIR}/src/main/resources"
+  "${CMAKE_CURRENT_BINARY_DIR}/src/main/java"
+  "${CMAKE_CURRENT_BINARY_DIR}/src/test/java"
+  )
+
+# find java
+find_package(Java)
+# find jni package
+find_package(JNI REQUIRED)
+
+# specify the output destination of cvcjni library
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/src/main/resources")
```

Comment:
Why do we want to change this? Wouldn't this change all targets from here on (also those not related to java)?

Reply:
Actually this is no longer needed. It is removed now